index.html

```diff
@@ -2565,28 +2565,20 @@ <h2>Representation Requirements</h2>
     </section>
 
     <section>
-      <h2>
-JSON
-      </h2>
+      <h2>JSON</h2>
 
       <p>
-This section sets out the requirements for producing and consuming <a>DID
-documents</a> that are in plain JSON (as indicated by a
-<code>contentType</code> of <code>application/did+json</code> in the resolver
-metadata).
+This section defines the <a>production</a> and <a>consumption</a> rules
+for the JSON <a>representation</a>.
       </p>
 
       <section>
         <h3>Production</h3>
 
         <p>
-A <a>DID document</a> MUST be a single <a data-cite="RFC8259#section-4">JSON
-object</a> conforming to [[!RFC8259]]. All entries of the <a>DID
-document</a> data model described in <a href="#data-model"></a>
-MUST be represented by using the entry key as the name of the
-member of the JSON object. The values of entries,
-including all extensions, are encoded in JSON [[RFC8259]] by mapping entry
-values to JSON types as follows:
+All data structures expressed by the <a href="#data-model">data model</a>
```

Comment:
```suggestion
The <a>DID document</a> and any DID document data structures expressed via the <a href="#data-model">data model</a>
```

Reply:
This change request implements https://www.w3.org/2021/02/16-did-topic-minutes.html#r01. @peacekeeper, I'm making the same changes to the other PRs... I assume this addresses your concern to the point where you'll approve merging the section?